documenters_aggregator/spiders/cchhs.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+import scrapy
+
+from datetime import datetime
+from pytz import timezone
+
+##########
+# To Do:
+#
+# (1) Step through past years--not sure why this doesn't work
+# (2) do we need to geocode location? implement classification? hard/fixed end time?
+##########
+
+
+class CchhsSpider(scrapy.Spider):
+    name = 'cchhs'
+    allowed_domains = ['http://www.cookcountyhhs.org']
+    start_urls = ['http://www.cookcountyhhs.org/about-cchhs/governance/board-committee-meetings/']
+    domain_root = 'http://www.cookcountyhhs.org'
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the `Open Civic Data
+        event standard <http://docs.opencivicdata.org/en/latest/data/event.html>`_.
+        """
+        for item in response.xpath("//a[@class='h2 accordion-toggle collapsed']"):
+            data = {
+                '_type': 'event',
+                'id': self._parse_id(item),
+                'name': self._parse_name(item),
+                'end_time': self._parse_end(item),
+                'all_day': self._parse_all_day(item)
+            }
+
+            aria_control = item.xpath("@aria-controls").extract_first()
+            item_uncollapsed = item.xpath("//div[@id='{}']//tbody//td[@data-title='Meeting Information']".format(aria_control))
+            for subitem in item_uncollapsed:
+                data.update({
```

Comment:
I'm not crazy about updating the same dict over and over here. But the flow is very clear for something relatively hairy.